source/_docs/automation.markdown

```diff
@@ -43,18 +43,51 @@ Actions are all about calling services. To explore the available services open t
 
 ### Automation initial state
 
-When you create a new automation, it will be enabled unless you explicitly add `initial_state: false` to it or turn it off manually via UI/another automation/developer tools. In case automations need to be always enabled or disabled when Home Assistant starts, then you can set the `initial_state` in your automations. Otherwise, the previous state will be restored.
+When you create a new automation, it will be enabled unless you explicitly add `initial_state: off` to it or turn it off manually via UI/another automation/developer tools. In case automations need to be always enabled or disabled when Home Assistant starts, then you can set the `initial_state` in your automations. Otherwise, the previous state will be restored.
 
 Please note that if for some reason Home Assistant cannot restore the previous state, it will result in the automation being enabled.
 
-```text
+```yaml
 automation:
 - alias: Automation Name
-  initial_state: false
+  initial_state: off
```

Comment:
Please stick to YAML 1.2 specifications for the documentation.

```suggestion
  initial_state: false
```

Reply:
Of course I'll do what you want, but could you please explain why? `off` seems to be accepted by the current YAML implementation, and even `'off'` is accepted by [`config_validation.boolean`](https://github.com/home-assistant/core/blob/c3b5bf74371499122cfb9af5ab2b9c35ab59f425/homeassistant/helpers/config_validation.py#L135-L148). And `off` is more natural here; even the description of the option uses the term "off".

Reply:
Because:
A) Not future proof. It isn't valid with the YAML 1.2 specification.
B) If is confusing for users: As `off` and `'off'` are not the same.
C) All documentation follows this standard.

Reply:
> It is confusing for users: As off and 'off' are not the same.

I would argue `false` is confusing. The entity's state is either `'on'` or `'off'`, not `true` or `false`. For good or bad, a decision was apparently made long ago that a binary entity's state is `'on'` / `'off'`, not `'true`'/`'false'`. But the documentation for `initial_state` says `false`??? _That_ is confusing.

And, as I said, the YAML & config validation implementations work together to make `'on'` & `on`, and `'off'` & `off` equivalent here.

I guess we'll have to agree to disagree.

Reply:
I guess so.